src/eventHandlers/pullRequest/index.ts

```diff
@@ -89,6 +91,19 @@ export const pullRequestHandle = async (
       )}.`,
     );
 
+    if (
+      checkPullRequestTitleForMergeCategory(
+        pullRequest.title,
+        getInput('MERGE_CATEGORY'),
+      ) === false
+    ) {
+      logInfo(
+        'Skipping auto-merge since the upgrade does not match merge category',
+      );
+
+      return;
```

Comment:
@merlinnot I couldn't find a good way of mocking the PR title.

Do you have some pointers for me on how I can go about testing this bit?

Reply:
I managed to get it tested!